### PR TITLE
Remove the enrollmentStorage and integrate with LedgerStorage

### PR DESCRIPTION
**Pull request by TrustHenry**
DB Instance is managed in one class and 
the required query operations are managed in LedgerStorage.

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -119,4 +130,107 @@ export class LedgerStorage extends Storages
...
+                {
+                    if (callback != null)
+                        callback(err);
+                    else
+                        return;
+                }
+            });
+        }
+        doPut();
+    }
```
Comment (Geod24) [inline src/modules/storage/LedgerStorage.ts:213]: Why does a foreach have to be 30 lines ?

Comment (Geod24) [reply src/modules/storage/LedgerStorage.ts:213]: Ah you are just moving the code

**Review by Geod24 — APPROVED** [review]

File: src/modules/storage/LedgerStorage.ts (hunk 1 above)
Comment (AndrejMitrovic) [reply src/modules/storage/LedgerStorage.ts:213]: More importantly, why is this recursive?

Comment (TrustHenry) [reply src/modules/storage/LedgerStorage.ts:213]: You're watching a feast of Asynchronous Bracelet.
If I find and write a good asynchronous library, I think the code will be simple.
I don't have enough time to apply it.

Comment (AndrejMitrovic) [reply src/modules/storage/LedgerStorage.ts:213]: I think this is the wrong place to write asynchronous code.

Comment (TrustHenry) [reply src/modules/storage/LedgerStorage.ts:213]: @AndrejMitrovic 
I didn't understand what you meant.

**Pull request merged**